### PR TITLE
specify that the scm guide is for self-hosted ghe only

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs

### CODE REVIEW

**Review by raghavjain3 — APPROVED** [review]

File: docs/semgrep-app/scm.md [hunk 1]
```diff
@@ -12,9 +12,13 @@ import MoreHelp from "/src/components/MoreHelp"
...
 Semgrep App's Team and Enterprise tiers support repositories hosted on **GitHub Enterprise (GHE)** and **GitLab Self Managed (GLSM)** plans. When referring to SCM tools, this term means explicitly GHE or GLSM plans in this document.
 
+:::note
+This document covers the enablement of features for GitHub Enterprise **self-hosted** plans. For users of GitHub Enterprise **Cloud** plans, see [Getting started with Semgrep App](/docs/semgrep-app/getting-started-with-semgrep-app).
+:::
+
 ## Prerequisites
 
-You need the following permissions to integrate Semgrep into GHE or GLSM servers:
+You need the following permissions to integrate Semgrep into GHE Self-Hosted or GLSM servers:
```
Comment (pabloest) [inline docs/semgrep-app/scm.md:21]:
```suggestion
You need the following permissions to integrate Semgrep into GHE Server or GLSM servers:

```

File: docs/semgrep-app/scm.md [hunk 2]
```diff
@@ -23,7 +27,7 @@ You need the following permissions to integrate Semgrep into GHE or GLSM servers
 
 Semgrep App requires PATs with assigned scopes. These scopes grant necessary permissions to the PAT and vary depending on the user's SCM.
 
-| GitHub Enterprise          | GitLab Self-Managed        |
+| GitHub Enterprise Self-Hosted          | GitLab Self-Managed        |
```
Comment (pabloest) [inline docs/semgrep-app/scm.md:30]:
```suggestion
| GitHub Enterprise Server          | GitLab Self-Managed        |

```

File: docs/semgrep-app/scm.md [hunk 3]
```diff
@@ -43,13 +47,13 @@ Integrate Semgrep into these custom source code management (SCM) tools by follow
 </div>
 
 3. Select your SCM provider.
-4. For **GitHub Enterprise**, follow these steps:
+4. For **GitHub Enterprise Self-Hosted**, follow these steps:
```
Comment (pabloest) [inline docs/semgrep-app/scm.md:50]:
```suggestion
4. For **GitHub Enterprise Server**, follow these steps:

```

File: docs/semgrep-app/scm.md [hunk 4]
```diff
@@ -43,13 +47,13 @@ Integrate Semgrep into these custom source code management (SCM) tools by follow
 </div>
 
 3. Select your SCM provider.
-4. For **GitHub Enterprise**, follow these steps:
+4. For **GitHub Enterprise Self-Hosted**, follow these steps:
     1. Create a PAT by following the steps outlined in this [guide to creating a PAT](https://docs.github.com/en/enterprise-server@3.1/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). Ensure that the PAT is **[created with the required scopes](../scm/#table-of-required-scopes-for-pats)**.
-    2. Enter the personal access token generated into the Access Token field.
-    3. Enter your GHE base URL into the SCM Base URL.
+    2. Enter the personal access token generated into the **Access Token** field.
+    3. Enter your GHE Self-Hosted base URL into the **SCM Base URL**.
```
Comment (pabloest) [inline docs/semgrep-app/scm.md:53]:
```suggestion
    3. Enter your GHE Server base URL into the **SCM Base URL**.

```

File: docs/semgrep-app/scm.md [hunk 5]
```diff
@@ -12,9 +12,13 @@ import MoreHelp from "/src/components/MoreHelp"
 
 Semgrep App's Team and Enterprise tiers support repositories hosted on **GitHub Enterprise (GHE)** and **GitLab Self Managed (GLSM)** plans. When referring to SCM tools, this term means explicitly GHE or GLSM plans in this document.
 
+:::note
+This document covers the enablement of features for GitHub Enterprise **self-hosted** plans. For users of GitHub Enterprise **Cloud** plans, see [Getting started with Semgrep App](/docs/semgrep-app/getting-started-with-semgrep-app).
```
Comment (pabloest) [inline docs/semgrep-app/scm.md:16]:
```suggestion
This document covers the enablement of features for GitHub Enterprise **Server** plans. For users of GitHub Enterprise **Cloud** plans, see [Getting started with Semgrep App](/docs/semgrep-app/getting-started-with-semgrep-app).

```

**Review by pabloest — APPROVED** [review]

**Comment by s-santillan** [comment]
@pabloest Thanks for clarifying the actual product name! Merging now.

**Pull request merged**